components/Checklist.js

```diff
@@ -0,0 +1,12 @@
+export default function Checklist({menuItems}, {menuName}) {
+    return (
+        <div>
+            <button type="button" class="checklist">{menuName}</button>
+            {menuItems.map((m) => (
+            <div>
+            <label>{m}</label>
+            <input type="checkbox" /><br /></div>
```

Comment:
Not sure why the `</br>` is here? Generally its best to leave anything for appearance to CSS, so if you want padding or margin, use css not a `br` tag.